artifact/src/main/scala/precog/SbtPrecog.scala

```diff
@@ -43,13 +40,15 @@ object SbtPrecog extends SbtPrecogBase {
 
   override def projectSettings =
     super.projectSettings ++
-    addCommandAlias("releaseSnapshot", "; project /; reload; checkLocalEvictions; +publish") ++
-    Seq(
-      githubOwner := "precog",
-      githubTokenSource := TokenSource.Environment("GITHUB_TOKEN") || githubTokenSource.value,
-
-      resolvers += Resolver.githubPackages("precog"),
-      resolvers += Resolver.githubPackages("slamdata"))   // TODO remove
+      addCommandAlias(
+        "releaseSnapshot",
+        "; project /; reload; checkLocalEvictions; +publish") ++
+      Seq(
+        githubOwner := "precog",
+        githubTokenSource := TokenSource.Environment("GITHUB_TOKEN") || githubTokenSource.value,
+        resolvers += Resolver.githubPackages("precog"),
+        resolvers += Resolver.githubPackages("slamdata")
```

Comment:
Do we still need this `slamdata` resolver?

Reply:
Ah, yes exactly - was just writing a comment on that :)